test/re_test.cc

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 #include <gtest/gtest.h>
-#include "re.h"
+#include "../src/re.h"
```

Comment:
I agree we should add tests for that, but I would like to get through more of this patch before I add even more new code.